src/components/ItemCard.js

```diff
@@ -0,0 +1,8 @@
+export default function ItemCard() {
+  return (
+    <li className="Item-card">
+      <img src="" alt="" />
```

Comment:
Not used code should not show up at all I think. Better you leave it out, if you aren't showing an actual image.

Reply:
True that! Thanks!